src/GafferSceneUI/OptionInspector.cpp

```diff
@@ -213,6 +214,16 @@ IECore::ConstObjectPtr OptionInspector::value( const GafferScene::SceneAlgo::His
 	return nullptr;
 }
 
+IECore::ConstObjectPtr OptionInspector::fallbackValue() const
+{
+	if( const auto defaultValue = Gaffer::Metadata::value( g_optionPrefix + m_option.string(), "defaultValue" ) )
```

Comment:
Would be worth preconstructing a `const InternedString g_defaultValue`, since InternedString construction does cost something.

Reply:
Fixed in ce7960d